src/jquery.ime.inputmethods.js

```diff
@@ -174,6 +174,10 @@
 			name: 'translitterointi',
 			source: 'rules/fi/fi-transliteration.js'
 		},
+                'got': {
+			name:  '𐌲𐌿𐍄𐌹𐍃𐌺𐌰 𐍂𐌰𐌶𐌳𐌰',
+			source: 'rules/got/got.js'
+                },
```

Comment:
You will need a language->input method name mapping in this file. Refer other examples. You can avoid the confusion by renaming the input method to something else other than language code